src/core/base.ts

```diff
@@ -8,7 +11,7 @@ export function Component<T extends Function>(selector: string, options: Element
         } else {
```

Comment:
Can we remove else block? We don't need it since error is thrown in other case

Reply:
No. It's a basic check.